.ci/jobs/bump-opbeans-node-mbp.yml

```diff
@@ -0,0 +1,41 @@
+---
+- job:
+    name: apm-agent-nodejs/bump-opbeans-node-mbp
+    display-name: Bump nodejs version
+    description: Bump nodejs version
```

Comment:
Should I change these to something that identifies "Opbeans Node ...", otherwise it may be pretty confusing when looking at:

<img width="618" alt="Screen Shot 2022-06-14 at 8 06 20 AM" src="https://user-images.githubusercontent.com/46866/173611673-d6bfef17-c931-413a-867d-0b3fc051c5ea.png">